lib/record_store/provider/dnsimple.rb

```diff
@@ -64,11 +64,14 @@ def secrets
       end
 
       def build_from_api(api_record, zone)
+        fqdn = api_record.name.present? ? "#{api_record.name}.#{zone}" : zone
+        fqdn = "#{fqdn}." unless fqdn.ends_with?('.')
```

Comment:
Seems like all Providers need to add a dot suffix to a string.  Consider promoting this operation to Provider.

Reply:
I don’t disagree but also out of scope - so not right now. :-)